### ISSUE TITLE
Ueberzurg as an image viewer 

### ISSUE THREAD

**Issue by strogiyotec** [dylanaraps/neofetch#1453]
Good day. I can't use custom images because my terminals (gnome and alacritty) don't display images  via w3m. That is why I use [ueberzurg](https://github.com/seebye/ueberzug). It works with [ranger](https://github.com/ranger/ranger) as well. Are you going to support ueberzurg as an alternative to w3m ?

**Comment by dylanaraps** [comment]
Duplicate of: https://github.com/dylanaraps/neofetch/issues/1172

I myself have no interest in adding this, however I'll accept patches which implement support. This issue sat open for a long while before I closed it. I'm closing this one too (see rationale in #1392).

I'm just waiting on patches basically.

**Comment by strogiyotec** [comment]
@dylanaraps got it , thank you
